apps/system/js/operator_variant_handler.js

```diff
@@ -309,7 +313,8 @@
           'ril.ims.passwd': 'password',
           'ril.ims.httpProxyHost': 'proxy',
           'ril.ims.httpProxyPort': 'port',
-          'ril.ims.authtype': 'authtype'
+          'ril.ims.authtype': 'authtype',
+          'ril.ims.mtu': 'mtu'
```

Comment:
add test in https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L82